### PR TITLE
frontend/send: update balance after sending a transaction

### PR DESCRIPTION
**Pull request by Beerosagos**

### CODE REVIEW

File: frontends/web/src/routes/account/send/send.tsx [hunk 1]
```diff
@@ -166,6 +166,18 @@ class Send extends Component<Props, State> {
           break;
         }
         break;
+      case 'backend':
```
Comment (thisconnect) [inline frontends/web/src/routes/account/send/send.tsx:169]: small nit: To me `backend` doesn't make sense as everything that is pushed is coming from the backend. (same reason I think we should not have `api/backend`)

maybe `account` would be more suitable?

Comment (thisconnect) [reply frontends/web/src/routes/account/send/send.tsx:169]: ignore, I forgot that it is already like that.. :P

File: frontends/web/src/routes/account/send/send.tsx [hunk 2]
```diff
@@ -166,6 +166,18 @@ class Send extends Component<Props, State> {
...
+        switch (data) {
+        case 'newTxs':
```
Comment (benma) [inline frontends/web/src/routes/account/send/send.tsx:171]: this is the wrong event, you want `syncdone` like in account.tsx.

**Comment by thisconnect** [comment]
The balance on the send view did not update after the send success message.
    
    Changed send component to listen to syncdone event and update the balance
    to show the correct remaining balance.

Could you ad something like this to the commit body?

Also please add a short note to the changelog.

File: frontends/web/src/routes/account/send/send.tsx (hunk 2 above)
Comment (Beerosagos) [reply frontends/web/src/routes/account/send/send.tsx]: Changed, thanks!

**Comment by Beerosagos** [comment]
Now should be ok

**Review by thisconnect — APPROVED** [review]
LGTM

File: frontends/web/src/routes/account/send/send.tsx [hunk 3]
```diff
@@ -166,6 +166,18 @@ class Send extends Component<Props, State> {
...
         }
         break;
+      case 'account':
+        switch (data) {
+        case 'syncdone':
+          if (this.props.code) {
+            accountApi.getBalance(this.props.code)
+              .then(newBalance => {
+                this.setState({ balance: newBalance });
+              }).catch(console.error);
```
[the remark this answers is not in the document]
Comment (Beerosagos) [reply frontends/web/src/routes/account/send/send.tsx]: Nice! Updated, thx

**Pull request merged**